src/Utilities/Enum.h

```diff
@@ -256,3 +256,10 @@ class MouseCursorHotSpotY
 		return false;
 	}
 };
+
+enum class IronCurtainAffects : char
```

Comment:
```suggestion
enum class IronCurtainAffects : unsigned char
```
Perhaps, for consistency?